app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -110,24 +115,41 @@ function initializeCommentsPage() {
         function successCb(response) {
           var reactionCount = thisButt.querySelector('.reactions-count');
           var reactionLabel = thisButt.querySelector('.reactions-label');
+          var heartFilledSvg = thisButt.querySelector('.reaction-icon.reacted');
+          var heartNotFilledSvg = thisButt.querySelector('.reaction-icon.not-reacted');
           if (response.result === 'create') {
             thisButt.classList.add('reacted');
+            thisButt.ariaPressed = true;
             if (reactionCount) {
               reactionCount.innerHTML = parseInt(reactionCount.innerHTML) + 1;
               reactionCount.classList.remove("hidden");
               if(parseInt(reactionCount.innerHTML) == 1) {
                 reactionLabel.innerHTML = "&nbsp;like"
+                heartFilledSvg.querySelector('title').textContent = "Like Comment: 1 like"
+                heartNotFilledSvg.querySelector('title').textContent = "Unlike Comment: 1 like"
```

Comment:
Since we're using `aria-pressed`, we should keep the accessible name of the button static throughout (i.e. never changing the name from 'like' to 'unlike').

There's a really great explanation of why in [this post on accessible toggle buttons](https://inclusive-components.design/toggle-button) (scroll to the "changing labels" heading 🙂 )

Reply:
@aitchiss Oh I see it now. Didn't bother reading the entire thing earlier 😅 .